Terminal.Gui/Core/View.cs

```diff
@@ -1186,7 +1188,8 @@ public virtual void Redraw (Rect bounds)
 		{
 			var clipRect = new Rect (Point.Empty, frame.Size);
 
-			Driver.SetAttribute (HasFocus ? ColorScheme.Focus : ColorScheme.Normal);
```

Comment:
In my opinion if `ColorScheme `is null and the `Driver `wasn't initialized then it should trigger an argument null exception instead by passing the null value because there will be used `Driver `functions in the views.

```cs

if (ColorScheme == null)
	throw new ArgumentNullException ("The Driver must be already created before using Redraw!");

```

Reply:
@tig can you give your opinion about this please.